### PR TITLE
nimble/transport: Fix syscfg definition names for uart_ll transport

### PR DESCRIPTION
**Pull request by bojanpotocnik**
Change `BLE_TRANSPORT_UART_*` in `hci_uart.c` to `BLE_TRANSPORT_UART_LL_*` to match definitions in `syscfg.yml`.

`uart_ll` was added by 80ef459c48dd2e938789013da1222438ffa2bf92 (https://github.com/apache/mynewt-nimble/pull/1399) as a copy of existing `uart` transport. `_LL` was added to `BLE_TRANSPORT_UART_*` definitions in `syscfg.yml`, but not in the `hci_uart.c` file.

Fixes https://github.com/apache/mynewt-nimble/issues/1438

### CODE REVIEW

**Review by andrzej-kaczmarek — APPROVED** [review]

**Comment by bojanpotocnik** [comment]
Thank you for approval, however I have no merge rights

**Comment by andrzej-kaczmarek** [comment]
I had to wait for CI to complete, merged now. Thanks!

**Pull request merged**
